### DISCUSSION TITLE
Release v5.0.0

### DISCUSSION THREAD

**Discussion by cameronkroeker** [Esri/arcgis-cookbook#384] [Announcements]
- Added support for ArcGIS Enterprise 11.3
- Added support for ArcGIS Video Server 11.3
- Added support for ArcGIS Pro 3.3
- Added support for ArcGIS License Manager 2024.0
- Added support for ArcGIS Insights 2023.3 and 2023.2
- Added arcgis-license-manager cookbook
- Added support for [Portal for ArcGIS Validation and Repair](https://support.esri.com/en-us/patches-updates/2023/portal-for-arcgis-validation-and-repair) tool 10.9.1 and 11.1
- Added recipes for WebGISDR export and import
- Updated third party cookbooks to latest versions
- Removed arcgis-desktop cookbook
- Dropped support for Chef Client 14

**Issues Addressed:**
- https://github.com/Esri/arcgis-cookbook/issues/380
- https://github.com/Esri/arcgis-cookbook/issues/372

<hr /><em>This discussion was created from the release <a href='https://github.com/Esri/arcgis-cookbook/releases/tag/v5.0.0'>Release v5.0.0</a>.</em>

**Comment by awittner** [comment]
@cameronkroeker - The [11.3 Functionality Matrix for ArcGIS Enterprise](https://www.esri.com/content/dam/esrisites/en-us/media/manuals/arcgis-enterprise-functionality-matrix-current.pdf) lists Notebook Server on Windows OS, in addition to Linux. v5.0 of cookbooks doesn't list a windows folder under Notebook Server Templates. The Matrix does recommend deploying Notebook Server on linux...but given the matrix listing - are there plans to provide windows templates for Notebook Server?

Noting: ArcGIS Notebook Server with Docker Desktop on Windows deprecation notice
__Version 11.3 will be the last version of ArcGIS Notebook Server to support Docker Desktop on Windows. Support for Docker Desktop on Windows will be fully removed in future releases. If you currently use Docker Desktop on Windows as a container engine for Notebook Server, it is recommended that you transition to an alternative container engine, such as Mirantis Container Runtime or Docker Engine from binaries, both of which support Windows Server 2022 with Windows containers._ 

We are running on Windows 2022, looking at Docker Engine.
